contracts/coordinator/test/mixins.ts

```diff
@@ -68,6 +68,8 @@ describe('Mixins tests', () => {
             takerAssetAmount: constants.ZERO_AMOUNT,
             makerFee: constants.ZERO_AMOUNT,
             takerFee: constants.ZERO_AMOUNT,
+            makerFeeAssetData: constants.NULL_BYTES,
+            takerFeeAssetData: constants.NULL_BYTES,
```

Comment:
Q: Are there any constraints on what assetData can be used for fees? Could one use `MultiAsset` for example?

Reply:
Yup, should work just fine. I don't think I have any explicit tests for MAPs, but I'll look into adding some. MAPs are kind of under-represented across all our tests.